README.md

```diff
@@ -152,11 +145,12 @@ Get started by running the "noop" transform that performs an identity operation
 [tutorial](data-processing-lib/doc/simplest-transform-tutorial.md) and associated 
 [noop implementation](transforms/universal/noop). 
 
-### Run a Data Pipeline on Local Ray
-Get started by building a data pipeline with our [example pipeline](./examples/) that can run on a laptop. To test this pipeline, you can download this repo as a zip file and get started. 
+### Run a Jupyter notebook on Local Ray cluster
+Get started by building a data pipeline with our [example pipeline](./examples/) that can run on a laptop. 
```

Comment:
Get started by building a Jupiter notebook executing a sequence of Transform executions with our [example pipeline](./examples/) that can run on a laptop.

Reply:
done, thanks